extensions/blocks/gif/gif.php

```diff
@@ -42,19 +42,26 @@ function jetpack_gif_block_render( $attr ) {
 	if ( isset( $attr['className'] ) ) {
 		array_push( $classes, $attr['className'] );
 	}
-	$classes = implode( $classes, ' ' );
+	$classes     = implode( $classes, ' ' );
+	$placeholder = sprintf( '<a href="%s">%s</a>', esc_url( $giphy_url ), esc_attr( $search_text ) );
 
 	ob_start();
 	?>
 	<div class="<?php echo esc_attr( $classes ); ?>">
 		<figure>
-			<div class="wp-block-jetpack-gif-wrapper" style="<?php echo esc_attr( $style ); ?>">
-				<iframe src="<?php echo esc_url( $giphy_url ); ?>"
-						title="<?php echo esc_attr( $search_text ); ?>"></iframe>
-			</div>
+			<?php if ( Jetpack_AMP_Support::is_amp_request() ) : ?>
+				<amp-iframe src="<?php echo esc_url( $giphy_url ); ?>" width="100" height="<?php echo intval( $padding_top ); ?>" sandbox="allow-scripts allow-same-origin" layout="responsive">
```

Comment:
How about using `absint` instead of `intval` here?

Reply:
Done in https://github.com/Automattic/jetpack/pull/11318/commits/550035495b9d8cf54b8ac417e107b6b8278b978d